### DISCUSSION TITLE
Specify Target Block/ Fee per vByte for the payout payment processor.

### DISCUSSION THREAD

**Discussion by Puffball101961** [btcpayserver/btcpayserver#4050] [Ideas / Feature Requests]
Feature Request
- Add a default target block option in the payment processors tab
- Allow the specification of a fee per vbyte when creating a payout using the Greenfield API. (https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Payouts_CreatePayoutThroughStore)

**Comment by bolatovumar** [comment]
> Allow the specification of a fee per vbyte when creating a payout using the Greenfield API.

Do you mean something like a suggested fee with which you would like to have the payout be processed?

**Comment by Kukks** [comment]
I don't think the second feature makes sense. Specifying the fee on an individual payout would mean that batching payouts would have no  way of knowing which fee to use and cause confusion for payout creation.


Working on the first option now

**Comment by Kukks** [comment]
https://github.com/btcpayserver/btcpayserver/pull/4065

**Comment by Puffball101961** [comment]
Didn't realise that the second feature would prevent transaction batching. Thanks for implementing the first feature.
